packages/gimme/components/sections/Portfolio/VaultRow.tsx

```diff
@@ -96,12 +82,10 @@ export function VaultRow(props: {
 				}
 			}
 		});
-	};
-
-	const onWithdraw = (): void => {
-		onSetVaultToWithdraw();
-		onSetAssetToWithdraw();
-		onSetAssetToReceive();
+		dispatchConfiguration({
+			type: 'SET_TOKEN_TO_RECEIVE',
+			payload: {...props.vault.token, chainID: props.vault.chainID, value: 0, balance: props.balance}
```

Comment:
Not sure yet but `balance` might be different here - underlying token balance instead of vault token balance